sdk/tables/azure-data-tables/test/ut/enum_operators_test.hpp

```diff
@@ -0,0 +1,21 @@
+#pragma once
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+// clang-format off
+#include "azure/data/tables/account_sas_builder.hpp"
+#include "azure/data/tables/tables_sas_builder.hpp"
+#include "test/ut/test_base.hpp"
+
+namespace Azure { namespace Data { namespace Test {
+  enum class TestEnum
+  {
+    Zero = 0,
+    One = 1,
+    Two = 2,
+    Three = 3,
+    All = ~0
```

Comment:
You might want to consider making the value of `Three` be something other than 3, since you're testing logical operators.

Reply:
i already have 0 and all , also those tests are for code coverage more than anything else, i mean what they test is will 1&1 equal 1 when cast to int ?